### ISSUE TITLE
Synaptics WBDI “fingerprint” was can not use

### ISSUE THREAD

**Issue by sunyink** [Nummer/Destroy-Windows-10-Spying#391]
The fingerprint module of the laptop can not be used, even after opening the “WbioSrvc” in the service.

**Comment by benzBrake** [comment]
Duplicate of #283

**Comment by sunyink** [comment]
It's true！ is ok~
Thank you~~

**Comment by sunyink** [comment]
Yes, thank you！~    the“Windows hello”is no longer compatible after this setting.But then I came across another problem.The “Windows hello installer“ flash out of the fingerprint program is still useless. It has been solved by me, and I leave it now.
run>regedit.msc
C**omputer configuration -- Windows settings -- security settings -- local strategy -- Security Options**
1.User account control: administrator approval mode for built-in administrator account
2.User account control: running all administrators with administrator approval mode.
The two items are all opened and restarted.
